Makefile

```diff
@@ -28,8 +28,12 @@ MODULE_OBJ = $(SRCDIR)/rebloom.o
 MODULE_SO = $(ROOT)/rebloom.so
 
 DEPS = $(ROOT)/contrib/MurmurHash2.o \
+	   $(ROOT)/contrib/xxhash.o \
```

Comment:
If possible, please use an existing hash (murmur or whatever..)

Reply:
From my understanding, it is used in the team.
Benchmarks show it is considerably faster.

Reply:
After running my own benchmark, MurmurHash came on top some I remove XXHash.

Reply:
xxhash is still here.. but I did some research and ppl claim it's faster, so... why not.